tests/integration/update.bats

```diff
@@ -312,3 +312,46 @@ EOF
     check_cgroup_value "cpu.rt_period_us" 900001
     check_cgroup_value "cpu.rt_runtime_us" 600001
 }
+
+@test "update devices [minimal transition rules]" {
+    [[ "$ROOTLESS" -ne 0 ]] && requires rootless_cgroup
+	# This test currently only makes sense on cgroupv1.
+    requires cgroups_v1
+
+	# Run a basic shell script that tries to write to /dev/null. If "runc
+	# update" makes use of minimal transition rules, updates should not cause
+	# writes to fail at any point.
+	jq '.process.args = ["sh", "-c", "while true; do echo >/dev/null; done"]' config.json > config.json.tmp
+	mv config.json{.tmp,}
+
+	# Set up a temporary console socket and recvtty so we can get the stdio.
+	TMP_RECVTTY_DIR="$(mktemp -d "$BATS_TMPDIR/runc-tmp-recvtty.XXXXXX")"
+	TMP_RECVTTY_PID="$TMP_RECVTTY_DIR/recvtty.pid"
+	TMP_CONSOLE_SOCKET="$TMP_RECVTTY_DIR/console.sock"
+	CONTAINER_OUTPUT="$TMP_RECVTTY_DIR/output"
+	("$RECVTTY" --no-stdin --pid-file "$TMP_RECVTTY_PID" \
+				--mode single "$TMP_CONSOLE_SOCKET" &>"$CONTAINER_OUTPUT" &) &
```

Comment:
I know this comes from `helpers.bats` but the double `&` is not really needed here, you'll be fine with something like
```shell
("$RECVTTY" ... ) &
```

IOW inner `&` is not needed.